### DISCUSSION TITLE
I must be stupid

### DISCUSSION THREAD

**Discussion by johnlattanzio** [community/community#80392] [New to GitHub]
So I _want_ to be a new CoPilot user but I am lost :-)
I have installed and begun a trial subs. When I try to use CoPilot (in Visual Studio) it tells me it wants me to log in via github. I say OK and it opens a new VS window that  seems to indicate that CoPilot is installed but when I return to my original notebook I am asked to log in.

Sorry for such a basic question!
John L

**Comment by MantasTech** [comment]
Hi John!

1. Log out of GitHub in all VS instances and close the IDE.
2. Reopen VS and log in to GitHub when prompted for Copilot.

If it didn't help, then just restart your device, you might be facing login loop issue. 

Hope it helped, take care!

**Reply by johnlattanzio** [reply]
Thanks. I had done that but...I realised it was opening a second VS so I just closed the first and all is fine :-)
Appreciate you taking the time to help me!
John L
